src/signals/incident-management/containers/IncidentDetail/index.test.js

```diff
@@ -16,9 +16,7 @@ import AttachmentViewer from './components/AttachmentViewer';
 import StatusForm from './components/StatusForm';
 import Detail from './components/Detail';
 import SplitNotificationBar from './components/SplitNotificationBar';
-import LocationPreview from './components/LocationPreview';
-//
-import statusList, { changeStatusOptionList } from '../../definitions/statusList';
+import LocationPreview from './components/LocationPreview'; import statusList, { changeStatusOptionList } from '../../definitions/statusList';
```

Comment:
`import` statements should be on their own line

Reply:
dit is een typo

Reply:
helaas